raiden/raiden_service.py

```diff
@@ -891,18 +891,22 @@ def start_mediated_transfer_with_secret(
                 amount=amount,
                 token_network_identifier=token_network_identifier,
                 payment_done=AsyncResult(),
-                secret=secret,
-                secret_hash=secret_hash,
             )
             self.targets_to_identifiers_to_statuses[target][identifier] = payment_status
 
+        # until we have full support create the secret here and ignore the provided hash
```

Comment:
I know what you're referring to when you say "until we have support", but others won't. Please clarify this comment